bin/boilerplate/_config.yml

```diff
@@ -16,8 +16,24 @@ title: "Lesson Title"
 # Life cycle stage of the lesson
 # See this page for more details: https://cdh.carpentries.org/the-lesson-life-cycle.html
 # Possible values: "pre-alpha", "alpha", "beta", "stable"
+#
+# Lessons that are going through the transition to the
+# Carpentries Workbench will go through 3 steps:
+# 'transition_step_1': notice indicating a new version
+# 'transition_step_2': notice encouraging to use new version
+# 'transition_step_3': notice indicating the lesson is deprecated,
+#                      with automated redirect
 life_cycle: "pre-alpha"
 
+# For lessons in the life stages in 'transition_step_1' or later:
+# - 'transition_url' holds the URL for the version of the lesson that
+#    uses the Workbench (needed for all 3 steps)
+# - 'transition_date' (in yyyy-mm-dd format) is the date when the lesson
+#    will transition to being deprecated. The date only needs to be decided
+#    when the lesson is in 'transition_step_2'.
+transition_url:
```

Comment:
Note: we have to make sure this transition URL is in the appropriate form before we move to step 3 since this acts like a 302 redirect.

e.g. https://example.com/page and https://example.com/page/ and https://example.com/page.html are different in the eyes of webcrawlers.

Reply:
I thought about this a little but I think a manual check will be better than over-engineering a technical solution given that I don't think we are completely set on what the future URL is going to look like.